README.md

```diff
@@ -16,12 +16,24 @@ SUBCOMMANDS:
 ```
 
 ### Example Usage
-
+#### Sway
 You can use pw-volume to bind multimedia keys to raise, lower, and mute volume.
 For instance, in Sway's config:
 
 ```
 bindsym XF86AudioRaiseVolume exec pw-volume change +2.5%
 bindsym XF86AudioLowerVolume exec pw-volume change -2.5%
 bindsym XF86AudioMute exec pw-volume mute toggle
-```
\ No newline at end of file
+```
+#### Waybar
+```
+"custom/pw-volume": {
```

Comment:
"`custom/pipewire`" more closely resembles the `pulseaudio` widget.

Reply:
Done.